sequencer.c

```diff
@@ -2654,6 +2654,29 @@ int sequencer_rollback(struct repository *r, struct replay_opts *opts)
 	return -1;
 }
 
+int sequencer_skip(struct repository *r, struct replay_opts *opts)
+{
+	switch (opts->action) {
+	case REPLAY_REVERT:
+		if (!file_exists(git_path_revert_head(r)))
+			return error(_("no revert in progress"));
+		break;
+	case REPLAY_PICK:
+		if (!file_exists(git_path_cherry_pick_head(r)))
+			return error(_("no cherry-pick in progress"));
+		break;
+	default:
+		BUG("the control must not reach here.");
+	}
+
+	if (rollback_single_pick(r))
+		return error(_("failed to skip the commit"));
+	if (!file_exists(git_path_seq_dir()))
```

Comment:
@dscho suggested using `is_directory` here (which I agree with).  That might have gotten lost in the discussion here: https://github.com/r1walz/git/pull/1#discussion_r288290246?  Or is there any other reason why we can't use that?

Reply:
Actually, everywhere else in the file, we are using `file_exists` so to maintain [ _consistency_](https://github.com/git/git/blob/74583d89127e21255c12dd3c8a3bf60b497d7d03/Documentation/CodingGuidelines#L31-L37) with the existing code I used `file_exists`. Should I change it here and now?

Reply:
I'd say consistency here is mostly about style, not necessarily which functions to use (though it never hurts to see what the existing code does already). 

I think at least `update_abort_safety_file` might want to use `is_directory` instead of `file_exists` as well, so it's better not to keep using the pattern. Fixing that would be out of scope for this patch series.